engine/storage/volume/src/main/java/org/apache/cloudstack/storage/volume/VolumeObject.java

```diff
@@ -678,6 +682,32 @@ protected void updateVolumeInfo(VolumeObjectTO newVolume, VolumeVO volumeVo, boo
         s_logger.debug(String.format("Updated %s from %s to %s ", volumeVo.getVolumeDescription(), previousValues, newValues));
     }
 
+    protected void updateResourceCount(VolumeObjectTO newVolume, VolumeVO volumeVo) {
```

Comment:
@sureshanaparti, we should rename `volumeVo` to avoid misleading with the class property `volumeVO`.

Reply:
```suggestion
    protected void updateResourceCount(VolumeObjectTO newVolume, VolumeVO newVolume) {
```